package.json

```diff
@@ -39,6 +39,7 @@
     "postinstall": "lerna bootstrap && yarn build && lerna link"
   },
   "devDependencies": {
+    "@geut/chan": "^2.1.1",
```

Comment:
Wtf

Reply:
🤣 util to add changes to the changelog following the keep a changelog pattern. It was already being used in the project, I just forgot to add it to the deps https://github.com/geut/chan/tree/master/packages/chan